### PR TITLE
Updates to L1T packer needed to simultaneously pack GT inputs and GMT/Calo ouputs

### PR DESCRIPTION
**Pull request by mulhearn**
This is the 81x version of #14323.

### CODE REVIEW

**Comment by mulhearn** [comment]
This PR passes internal L1T integration tests and bitwise reproduces L1T in CMSSW_806 (i.e. makes no change to L1T algorithms).

**Comment by mulhearn** [comment]
+1

File: EventFilter/L1TRawToDigi/python/gtStage2Raw_cfi.py [hunk 1]
```diff
@@ -4,7 +4,10 @@
     "L1TDigiToRaw",
     Setup = cms.string("stage2::GTSetup"),
 #    TowerInputLabel = cms.InputTag("simCaloStage2Digis"),
-    InputLabel = cms.InputTag("simGtStage2Digis"),
+    GtInputLabel = cms.InputTag("simGtStage2Digis"),
```
Comment (davidlange6) [inline EventFilter/L1TRawToDigi/python/gtStage2Raw_cfi.py]: @mulhearn - these aren't available for data... whats the timescale for making the real digis?

File: EventFilter/L1TRawToDigi/src/implementations_stage2/GTSetup.cc [hunk 2]
```diff
@@ -70,17 +75,17 @@ namespace l1t {
                if (fed == 1404) {
```
Comment (davidlange6) [inline EventFilter/L1TRawToDigi/src/implementations_stage2/GTSetup.cc]: better if this magic number ended up in the FedRawData header file that has these numbers already

File: EventFilter/L1TRawToDigi/python/gtStage2Raw_cfi.py (hunk 1 above)
Comment (davidlange6) [inline EventFilter/L1TRawToDigi/python/gtStage2Raw_cfi.py]: @mulhearn clariifies - its the digi->raw here..

**Pull request merged**